src/PowerShell/PowerShellModuleDetector.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Microsoft.Azure.Functions.PowerShellWorker.Utility;
+using System;
+using System.Linq;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Functions.PowerShellWorker.PowerShell
+{
+    using LogLevel = WebJobs.Script.Grpc.Messages.RpcLog.Types.Level;
+
+    internal class PowerShellModuleDetector
+    {
+        public static bool IsPowerShellModuleLoaded(System.Management.Automation.PowerShell _pwsh, ILogger _logger, string moduleName)
+        {
+            // Search for the external DF SDK in the current session
```

Comment:
Please update this comment as the external DF SDK is one of the possible modules.

Reply:
Done